server/api/sendGrid.js

```diff
@@ -5,15 +5,21 @@ const config = require("../config/config.js")
 
 sgMail.setApiKey(config.SENDGRID_API_KEY)
 
-function sendAnEmail(recipient, sender, subject, text) {
+function sendAnEmail(req, res) {
     const msg = {
-        to: recipient,
-        from: sender,
-        subject: subject,
-        text: text,
-        html: `<strong>${text}</strong>`,
+        to: req.body.recipient,
```

Comment:
- [ ] what if the sender needs to send for more than one recipient?

Reply:
you can do that from the front end but it is better if he fix it

Reply:
you can actually, just simply add the first parameter as an array of strings